integration/tests/cook/test_basic.py

```diff
@@ -760,3 +762,52 @@ def test_basic_docker_job(self):
         self.assertEqual(resp.status_code, 201)
         job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
         self.assertEqual('success', job['instances'][0]['status'])
+
+    def get_executor(agent_state, instance):
+        for framework in agent_state['frameworks']:
+            for executor in framework['executors']:
+                if executor['id'] == instance['executor_id'][0]:
+                    return executor
+
+    def test_docker_port_mapping(self):
+        job_uuid, resp = util.submit_job(self.cook_url,
+                                         command='python -m http.server 8080',
+                                         ports=2,
+                                         container={'type': 'DOCKER',
+                                                    'docker': {'image': 'python:3.6',
+                                                               'network': 'BRIDGE',
+                                                               'port-mapping': [{'host-port': 0,
+                                                                                 'container-port': 8080},
+                                                                                {'host-port': 1,
```

Comment:
Can we add comments about why `host-port` values are 0 and 1?

Reply:
done